AKS-Hybrid/deploy-gpu-node-pool.md

```diff
@@ -62,12 +62,37 @@ Install the Azure Stack HCI, version 23H2 operating system locally on each serve
 
 ### Step 2: uninstall the NVIDIA host driver
 
-On each host machine, navigate to **Control Panel > Add or Remove programs**, uninstall the NVIDIA host driver, then reboot the machine. After the machine reboots, confirm that the driver was successfully uninstalled. Open an elevated PowerShell terminal and run the following command:
+Open a remote powershell session to each host, or run the following local in Powershell. Start by uninstalling the NVIDIA host driver, then reboot the machine. After the machine reboots, confirm that the driver was successfully uninstalled:
+
+```powershell
+PNPUTIL /enum-drivers
+```
+
+Open an elevated PowerShell prompt and run the following command:
 
 ```powershell
 Get-PnpDevice  | select status, class, friendlyname, instanceid | findstr /i /c:"3d video" 
 ```
 
+You should see the installed drivers in the **PNPUTIL** output. If the **Provider Name** is listed as **NVIDIA Corporation**, that is the driver you need to target for uninstalling. Make a note of the **Published Name**, as you must use that in the next command:
+
+```output
+Published Name:     oem15.inf
+Original Name:      nvlwswi.inf
+Provider Name:      NVIDIA
+Class Name:         Display
+Class GUID:         {4d36e968-e325-11ce-bfc1-08002be10318}
+Driver Version:     03/05/2024 31.0.15.5178
+Signer Name:        Microsoft Windows Hardware Compatibility Publisher
+```
+
+Run the following command in your Powershell session, and replace `.\oem1.inf` with the value in **Published Name** from the previous **PNPUTIL Enum-Devices** output:
+
+```powershell
+pnputil /delete-driver .\oem1.inf /uninstall /reboot
```

Comment:
We'd better replace `.\oem1.inf` to something like `<oem#.inf>`, avoiding someone copy, paste, run without noticing your guideline above.

Reply:
I call this out on line 89 and suggest that you replace the value with a value from the output.

Reply:
Yes. But `oem1.inf` is a real driver. If someone just copy, paste and run without replacing the value, a driver will be removed unexpectedly. By given `<oem#.inf>` in sample code, the command with fail with noop to system. It's safer.